x-pack/test/functional/apps/saved_objects_management/feature_controls/saved_objects_management_security.ts

```diff
@@ -262,7 +262,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
           kibana: [
             {
               feature: {
-                visualize: ['all'],
+                visualize: ['minimal_all'],
```

Comment:
Because as I have it, Reporting is included in the `all` privileges. Tests that set up a user with `all` privileges for the visualize editor previously did not automatically gain access to the Management page, but they will now since privileges for Reporting automatically grants that access.

Reply:
I'm new to the Reporting feature so I tested this manually to make sure I fully understood.

So here is what a user with All access to the Dashboard feature (including the the Reporting sub-feature privileges) sees:

![image](https://user-images.githubusercontent.com/5295965/115301414-2880da00-a12f-11eb-90d0-69f3274cacdc.png)

If they navigate to a URL to inspect the object (on the SOM page), instead of seeing an "App not found" page, they see this:

![image](https://user-images.githubusercontent.com/5295965/115302370-5ca8ca80-a130-11eb-92fc-345ffd01e260.png)

So, your change to this integration test makes sense, though it might be a good idea to add some comments here as to why this will get a different result from `visualize: ['all']`.

-----

That aside: I don't love the fact that this sub-feature privilege implicitly/silently grants access to the Stack Management page. I can guess that users won't necessarily expect this behavior if they haven't given a user access to any of the Management features:

![image](https://user-images.githubusercontent.com/5295965/115301620-74338380-a12f-11eb-903d-8f66acb24d9f.png)

AFAIK, Reporting is the first sub-feature privilege that will behave this way.

I can think of two improvements for this:

1. Add a `privilegesTooltip` for each Reporting sub-feature privilege (similar to the tooltip on the Dev Tools feature in the screenshot above) that explains this will grant access to Stack Management
   * Note: currently, this is only an option for `KibanaFeatureConfig`, not for `SubFeatureConfig` -- we would have to make some updates to the `features` plugin to allow for this
2. Add a `reserved` Reporting feature in the Stack Management category (similar to the one for Stack Monitoring in the screenshot above) that explains if a user has access to any Reporting sub-feature privileges, they will also gain access to that Stack feature

WDYT @tsullivan ?

Reply:
> Add a reserved Reporting feature in the Stack Management category (similar to the one for Stack Monitoring in the screenshot above) that explains if a user has access to any Reporting sub-feature privileges, they will also gain access to that Stack feature

Good idea, will do

Reply:
On second thought: this is the same behavior that Search Sessions does, and there isn't a call-out in the section of privileges to the Management app there. Later, there may be other plugins or features that provide access to Management.

One thing that could be improved: the documentation should explicitly state that users with privilege to reporting features will have access to manage their own reports.